controllers/provisioners/eks/update_test.go

```diff
@@ -738,9 +745,10 @@ func TestUpdateWithLatestAmiID(t *testing.T) {
 	}
 
 	// Setup Latest AMI
-	ig.GetEKSConfiguration().Image = "latest"
-	ssmMock.latestAMI = testLatestAmiID
-
+	ssmMock.parameterMap = map[string]string{
+		"/aws/service/eks/optimized-ami/1.18/amazon-linux-2/recommended/image_id":                    testLatestAmiID,
+		"/aws/service/eks/optimized-ami/1.18/amazon-linux-2/amazon-eks-node-1.18-v20220226/image_id": testCustomAmi,
+	}
```

Comment:
Do we want to add cases for Windows/Bottlerocket, or do you feel it would be redundant?
Might be nice since it if this breaks somehow we would be in trouble.

Same for the other tests